airflow/dag_processing/manager.py

```diff
@@ -1129,7 +1129,6 @@ def _kill_timed_out_processors(self):
                 )
                 Stats.decr('dag_processing.processes')
                 Stats.incr('dag_processing.processor_timeouts')
-                # TODO: Remove after Airflow 2.0
```

Comment:
Can we TODO rmove on 3.0?

Reply:
I don't mind adding it, but it's an interesting question. From the (philosophical) discussion above, it seems like we're going to try very hard to not release a major/breaking version for quite some time, so I'm not sure we should leave TODOs for 3.0 yet (or ever?). WDYT from that perspective? also CC @potiuk

Reply:
I proposed neutral wording.

Reply:
Ok. I will merge as it is for now to not keep it waiting. It's good as it is now.